src/query/examples/concepts-queries-deduping-caching/craco.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path')
+const resolveFrom = require('resolve-from')
+
+module.exports = {
+  webpack: {
+    resolve: {
+      alias: {
+        '@reduxjs/toolkit/query': path.resolve(__dirname, '../../'),
+        '@reduxjs/toolkit': path.resolve(__dirname, '../../../'),
+        react: resolveFrom(
+          path.resolve(__dirname, '../../../node_modules'),
+          'react'
+        ),
+        'react-dom$': resolveFrom(
+          path.resolve(__dirname, '../../../node_modules'),
+          'react-dom'
+        ),
+        'eslint-plugin-prettier': resolveFrom(
+          'node_modules',
+          'eslint-plugin-prettier'
+        ),
+        'babel-jest': resolveFrom('node_modules', 'babel-jest'),
```

Comment:
Unfortunately, this doesn't work - even with specifying the resolutions. There is a conflict here between this and the babel-jest dep in the root for rtk itself. This is why `SKIP_PREFLIGHT_CHECK=true` is on :disappointed: